### DISCUSSION TITLE
Could someone export my project for me?

### DISCUSSION THREAD

**Discussion by prodDatriX** [SebiAi/custom-nothing-glyph-tools#22] [Q&A] [answered]
Hello, I have problems with Nothing Glyph Tools and I'm so desperate after 2 hours of trying to fix it.. I've got finished audacity project with labels, so I want to ask if someone could help me and export that for me, so I can use it without having to use python and this stuff.. My discord is: prod.datrix

**Comment by SebiAi** [comment, marked as answer]
Hi. Did you install python manually? If so use the `Install-Dependencies.bat` script to install it correctly.

**Reply by prodDatriX** [reply]
I have also problem with my audacity that I can't export it to ogg through Opus.. I select Opus file in export tab, i rename it from .opus to .ogg, and when i export it, it is still named .opus.. That is my first problem.. Second problem is with that python wich i will try how you said.. Thanks for helping me..

**Comment by SebiAi** [comment]
> I have also problem with my audacity that I can't export it to ogg through Opus.. I select Opus file in export tab, i rename it from .opus to .ogg, and when i export it, it is still named .opus.. That is my first problem.. Second problem is with that python wich i will try how you said.. Thanks for helping me..

For the exporting part: Export as Opus with the opus extension and then change the extension to ogg manually. [Make sure that you can see extensions in the file explorer](https://www.howtogeek.com/205086/beginner-how-to-make-windows-show-file-extensions/) or the rename won't work.

**Reply by prodDatriX** [reply]
I know that i have on extensions in file name so i can see .exe .txt and this stuff. My problem is that I export project -> I select "Opus Files" as file, rename extension in name from .opus to .ogg, and when i click in space or hit enter, it automatically changes it back to .opus.. I don't know if it is because I have newest version of audacity and exporting works pretty differently than before.. 
![image](https://github.com/SebiAi/custom-nothing-glyph-tools/assets/160472109/ada038b2-4a43-48fe-8984-ae5cf9cea4f1)
![image](https://github.com/SebiAi/custom-nothing-glyph-tools/assets/160472109/971a7849-d846-401a-aacc-d6b8422c1089)
![image](https://github.com/SebiAi/custom-nothing-glyph-tools/assets/160472109/3e80638f-cddb-45a3-953d-433b0f0b6a49)

**Comment by SebiAi** [comment]
The `opus` extension is expected.
1. Export as Opus with the wrong extension
2. Change the file extension with the file explorer to `ogg`

**Reply by prodDatriX** [reply]
I downloaded older version of audacity and it works in export tab now.. But problem is that it can't find avformat.dll.. I downloaded the zip file, extracted like in readme, i select the .dll file but it brings back the export tab and nothing happens.. I also tried to download that avformat.dll from web separatly and still doesn't work.. FFmpeg should be installed bcs of Install-Dependencies.bat..

**Comment by SebiAi** [comment]
> I downloaded older version of audacity and it works in export tab now.. But problem is that it can't find avformat.dll.. I downloaded the zip file, extracted like in readme, i select the .dll file but it brings back the export tab and nothing happens.. I also tried to download that avformat.dll from web separatly and still doesn't work.. FFmpeg should be installed bcs of Install-Dependencies.bat..

Use the new Audacity version and rename the extension. That works fine multiple people have done it.

Ffmpeg is indeed installed but not the shared libraries (avformat.dll for example).

**Reply by prodDatriX** [reply]
I'll contact you on discord if you are okay with it, I think it would be faster to comunicate..
